glossary.yml

```
@@ -10110,4 +10110,85 @@
     term: "non-parametric (statistics)"
     def: >
       A branch of statistical tests which do not assume a known distribution of the population which the samples were taken from (Kruskal-Wallis and Dunn test are examples of non-parametric tests).
-     
+- slug: artificial_intelligence
+  ref:
+    - nlp
+    - ma
```

```suggestion
    - ml
```
Did you mean machine learning?